templates/strings.js

```diff
@@ -17,7 +17,11 @@
                 required: '[% loc('Please enter your email') %]',
                 email: '[% loc('Please enter a valid email') %]'
             },
-            password: '[% loc('Please enter a password') %]'
+            password: {
+                required: '[% loc('Please enter a password') %]',
+                short: '[% loc('Please enter a password at least %d characters long') %]',
+                common: '[% loc('Please choose a less commonly-used password') %]'
```

Comment:
This line is in the wrong commit :)

Reply:
Actually it's not needed at all! I initially thought this error would need to be present in the app code, but it's provided by the server if the password is too short.

Reply:
Oh yes, and I even knew that because I tested to see if JSON parsed just a string okay :)